lib/simp/cli/commands/bootstrap.rb

```diff
@@ -200,46 +200,29 @@ def self.run(args = [])
       pupcmd += " --pluginsync"
     end
 
-    # The final tagged run is pupmod, standalone.  It is isolated to mitigate b0rked
-    # puppet runs caused by the inevitable restart of the puppetserver service during
-    # its application to the system.
-    pupruns = [
-      'pki,stunnel,concat',
-      'firstrun,concat',
-      'rsync,concat,apache,iptables',
-      'user',
-      'group',
-      'pupmod'
-    ]
-
-    # Begin tagged runs, against 8150.
-    puts "Beginning Puppet agent runs ..."
-    pupruns.each do |puprun|
-      puts "... with tag#{puprun.include?(',') ? 's' : ''} '#{puprun}'"
-      linecounts << track_output("#{pupcmd} --tags #{puprun} 2> /dev/null", '8150')
-    end
-
+    # Firstrun is tagged and run against the default puppetserver port, 8150.
+    # This run will configure puppetserver and puppetdb; all subsequent runs
+    # will run against the conifgured masterport.
+    puts "Running puppet agent, with tags pupmod,simp\n"
+    linecounts << track_output("#{pupcmd} --tags pupmod,simp 2> /dev/null", '8150')
```

Comment:
Are we sure that nothing that was removed here benefited from the `fixfiles -f relabel` below?